sdk/keyvault/keyvault-certificates/CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Release History
 
+## 4.0.0-preview.9 (2019-11-04)
+
+- Fixed parameter names and access modifiers to be on par of other language implementations.
+- Updated dependencies.
+- Cleaned up the exported types.
```

Comment:
This point doesnt tell the user how they will be affected. Either remove it or add more details. I would suggest removing it.